assets/cubyz/shaders/skybox/star.vs

```diff
@@ -0,0 +1,16 @@
+#version 430
+
+in vec3 vPos;
+in vec3 vColor;
+
+uniform mat4 projectionMatrix;
+uniform mat4 viewMatrix;
+uniform mat4 modelMatrix;
+
+out vec3 color;
+
+void main() {
+	gl_Position = projectionMatrix*viewMatrix*modelMatrix*vec4(vPos, 1);
```

Comment:
Also, please precompute `projectionMatrix*viewMatrix` on the CPU, to avoid repeating this calculation 100000 times.